esmvaltool/_recipe.py

```diff
@@ -425,10 +425,15 @@ def _get_default_settings(variable, config_user, derive=False):
 
     # Set up downloading using synda if requested.
     if config_user['synda_download']:
-        # TODO: make this respect drs or download to preproc dir?
+        # TODO: make this respect drs
         download_folder = os.path.join(config_user['preproc_dir'], 'downloads')
         settings['download'] = {
             'dest_folder': download_folder,
+            'variable': variable,
```

Comment:
Instead of passing `'variable', 'rootpath', 'drs', 'to_repository'`, simply use that information here to determine the correct `download_folder` and pass a `dest_folder` argument to the `download` function that it will actually use.